content/en/user-guide/aws/lambda/index.md

```diff
@@ -167,6 +166,11 @@ LocalStack now supports a new event rule engine for [Lambda event filtering](htt
 You can [configure]({{< ref "configuration" >}}) `EVENT_RULE_ENGINE=java` (preview) to use the AWS [event-ruler](https://github.com/aws/event-ruler), which offers better parity.
 {{< /callout >}}
 
+{{< callout >}}
+LocalStack now supports a Event Source Mapping (ESM) provider.
```

Comment:
Let's call it a new implementation :)

It's also technically not a provider (just a feature flag) and I think "new implementation" is clearer anyway.